corehq/toggles.py

```diff
@@ -784,6 +795,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_PRODUCT_PATH,
     [NAMESPACE_DOMAIN],
     0.1,
+    always_disabled=['hsph-betterbirth'],
```

Comment:
it is indeed just the domain, i checked locally as well because i also get confused